ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -538,7 +538,10 @@
                 "\n{",
                 "\n    static ApplicabilityBuilder ScopeApplicabilityBuilder(ApplicabilityBuilder builder) =>",
                 "\n        builder.ForScope<NominalCashflow>(s => s.WithApplicability<CreditDefaultRiskNominalCashflow>(x => x.Identity.AmountType == AmountTypes.CDR && x.Identity.Id.AocType == AocTypes.CF)",
-                "\n                                                .WithApplicability<AllClaimsCashflow>(x => x.Identity.AmountType == AmountTypes.CDR));",
+                "\n                                                .WithApplicability<AllClaimsCashflow>(x => x.Identity.AmountType == AmountTypes.CDR)",
+                "\n                                                .WithApplicability<ProjectionBopNominalCashflow>(x => x.GetStorage().GetShift(x.Identity.Id.ProjectionPeriod) >= MonthInAYear",
+                "\n                                                                                        && x.Identity.Id.AocType == \"BOP\" ",
```

Comment:
Do not hardcode the values but use the consts

Reply:
Removed hardcoding.